cyberint-argos/cyberint-workflow.xml

```diff
@@ -14,73 +15,83 @@
         <!-- Initialize the risingValue filter for the first run. -->
         <Initialize path="/bookmark" value="${time() - 604800000}" />
 
-        <!-- Prepare the start date filter, based on the current bookmark -->
-        <FormatDate pattern="yyyy-MM-dd'T'HH:mm:ss" time="${/bookmark}" timeZone="UTC" savePath="/start_date" />
+        <!-- Prepare the start time filter, based on the current bookmark -->
+        <FormatDate pattern="yyyy-MM-dd'T'HH:mm:ss" time="${/bookmark}" timeZone="${/timeZone}" savePath="/startTime" />
 
-        <!-- Prepare the end date filter, which is the current time. -->
-        <FormatDate pattern="yyyy-MM-dd'T'HH:mm:ss" timeZone="UTC" savePath="/end_date" />
+        <!-- Prepare the end time filter, which is the current time. -->
+        <FormatDate pattern="yyyy-MM-dd'T'HH:mm:ss" timeZone="${/timeZone}" savePath="/endTime" />
 
-        <!-- Get Alerts -->				
-        <CallEndpoint url="${/hostname}/alert/api/v1/alerts" method="POST" savePath="/get_alerts" >
-            <RequestHeader name="Cookie" value="access_token=${/api_key}; Path=/;" />
+        <!-- Get Alerts -->
+        <CallEndpoint url="${/hostName}/alert/api/v1/alerts" method="POST" savePath="/getAlerts" >
+            <RequestHeader name="Cookie" value="access_token=${/apiKey}; Path=/;" />
             <RequestBody type="application/json" encoding="UTF-8">
                 {
                     "page": 1,
-                    "size": ${/page_size},
+                    "size": ${/pageSize},
                     "include_csv_attachments_as_json_content": true,
                     "filters":
-                        {   
-                            "status": 
+                        {
+                            "status":
                                 [
                                     ${/status}
                                 ],
-                            "severity": 
+                            "severity":
                                 [
                                     ${/severity}
                                 ],
-                            "created_date": 
+                            "modification_date":
                                 {
-                                    "from": "${/start_date}",
-                                    "to": "${/end_date}"
+                                    "from": "${/startTime}",
+                                    "to": "${/endTime}"
                                 }
                         }
                 }
             </RequestBody>
         </CallEndpoint>
 
         <!-- Handle Errors -->
-        <If condition="/get_alerts/status_code != 200">
-            <If condition="/get_alerts/status_code = 403" >
+        <If condition="/getAlerts/status_code != 200">
+            <If condition="/getAlerts/status_code = 403" >
                 <Abort reason="The user or team account does not have access to the endpoint/feature/resource." />

```

Comment:
Is it alright that the logs are below the `Abort`?

Reply:
fixed